File: assignment/daysoftheweek.py
reply = int(input("Enter a number within 1-7: "))

match (reply) :
	case 1:
		print("Monday")

	case 2:
		print("Tuesday") 

	case 3:
		print("Wednesday")
		
	case 4:
		print("Thursday")

	case 5:
		print("Friday")

	case 6:
		print("Saturday")

	case 7:
		print("Sunday")
	case _:
		print("Not found, input number from 1-7")

File: assignment/DaysOfTheWeek.java
import java.util.Scanner;

public class DaysOfTheWeek {

	public static void main(String... args) {

		Scanner input = new Scanner(System.in);

		System.out.print("Enter a number within 1-7: ");
		int reply = input.nextInt();

		switch (reply) {
		case 1:
			System.out.print("Monday"); break;

		case 2:
			System.out.print("Tuesday"); break;

		case 3:
			System.out.print("Wednesday"); break;
		
		case 4:
			System.out.print("Thursday"); break;

		case 5:
			System.out.print("Friday"); break;

		case 6:
			System.out.print("Saturday"); break;

		case 7:
			System.out.print("Sunday"); break;

		case _:
			System.out.print("input the right number, arghhh!"); break;

	}
    }
}











